ingesters/kafka.md

```diff
@@ -88,7 +88,9 @@ As with most ingesters, each consumer may also specify [preprocessors](/ingester
 	Synchronous=true
 	Source-Header=SRC #A custom feeder is putting its source IP in the header named "SRC"
 	Batch-Size=256 #get up to 256 messages before consuming and pushing
+    # prefer sticky but present sticky and roundrobin as options we support
```

Comment:
```suggestion
        # this config sets sticky as the preferred strategy but also offers roundrobin
```

I think it is better to avoid "options we support" in the comment. Minor detail, but I don't want to confuse people about whether we (Gravwell) support the `range` option.